pages/index.tsx

```diff
@@ -72,6 +72,7 @@ export default function Home({ deviceType, singleItemDetail }: Props) {
     url,
     dataFetcher,
     {
+      isPaused: () => !coordinatesAndEventType,
```

Comment:
Mantikli geldi bana. @EmirBoyaci senin cozum onerin neydi?

Reply:
Diğer bir PR'da setURL yaparken undefined kontrolü yapılmış. Buda bir çözüm ama Home gövdesinde çok fazla setURL olduğu için hepsine check yapmak gerekebilir. O yüzden çözümü SWR tarafında yaparsak daha iyi olabilir.

Reply:
ama url'in undefined değerlerle set olması sağlıklı değil gibi. ikisin merge edebiliriz belki ?

Reply:
@absolutezero13 Evet o konuda haklısın. Aslında gereksiz useEffect kullanımı var. 2sini birleştirirsek yukarıdada check var zaten, o yüzden sorun çözülmüş olur.
![image](https://user-images.githubusercontent.com/23298031/218081796-43b70cdb-1951-40e4-be3a-1757898f91f5.png)